src/vdaf/poplar1.rs

```diff
@@ -531,8 +518,9 @@ where
         let level = get_level(agg_param)?;
 
         // Derive the verification randomness.
-        let mut verify_rand_prng: Prng<I::Field, _> =
-            Prng::from_seed_stream(P::seed_stream(&verify_param.rand_init, nonce));
+        let mut p = P::init(agg_key);
+        p.update(nonce);
+        let mut verify_rand_prng: Prng<I::Field, _> = Prng::from_seed_stream(p.into_seed_stream());
```

Comment:
Should we modify `P::seed_stream` to take a `[u8; L]` rather than a `Seed<L>`? I made a similar change to `P::init` to avoid a copy. (See below.)